src/App.css

```diff
@@ -6,18 +6,73 @@
 @tailwind utilities;
 
 @layer utilities {
-  .image-mask {
-    -webkit-mask-image: linear-gradient(
-      0deg,
-      rgba(0, 0, 0, 0) 5%,
-      rgba(0, 0, 0, 1) 30%
-    );
-    mask-image: linear-gradient(
-      0deg,
-      rgba(0, 0, 0, 0) 5%,
-      rgba(0, 0, 0, 1) 30%
-    );
-    -webkit-mask-repeat: no-repeat;
-    mask-repeat: no-repeat;
+  .perspective {
+    perspective: 700px;
+    perspective-origin: center center;
+    transform: rotateY(25deg);
   }
+
+  .cube-container {
+    perspective: 1000px;
+  }
+
+  .cube {
+    transform-style: preserve-3d;
+    animation: spin 300s infinite linear;
+  }
+
+  .cube:hover {
+    animation-play-state: paused;
+  }
```

Comment:
1. Faire une class `.preserve-3d`
2. Déplacer l'animation dans `tailwind.config`
3. Faire une class `.pause-animation` et l'utiliser [comme d'habitude](https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active)